packages/editor/src/store/selectors.js

```diff
@@ -1671,6 +1671,24 @@ export function __experimentalGetDefaultTemplateTypes( state ) {
 	return getEditorSettings( state )?.defaultTemplateTypes;
 }
 
+/**
+ * Returns the default template part areas.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {Array} The template part areas.
+ */
+export const __experimentalGetDefaultTemplatePartAreas = createSelector(
+	( state ) => {
+		const areas = getEditorSettings( state )?.defaultTemplatePartAreas;
+		const areasWithIcons = areas?.map( ( item ) => {
+			return { ...item, icon: getTemplatePartIconByArea( item.area ) };
+		} );
+		return areasWithIcons;
```

Comment:
Nitpick: you don't need to declare `areasWithIcons`, return the result from `areas?.map`.

Reply:
updated